reference/enterprise-api-ref/web-rbac.markdown

```diff
@@ -0,0 +1,264 @@
+---
+layout: default
+title: Web RBAC API
+published: true
+tags: [reference, enterprise, API, settings, RBAC]
+---
+Web RBAC API for managing role based access control settings.
+
+## Get all permissions list
+
+**URI:** https://hub.cfengine.com/api/rbac
+
+**Method:** GET
+
+**Example request (curl):**
+
+```
+curl -k --user <username>:<password> \
```

Comment:
I cry every time I see us do this in docs. I can't wait until we automate the docs for APIs.